core/src/main/java/org/elasticsearch/search/sort/FieldSortBuilder.java

```diff
@@ -35,8 +52,6 @@
 
     private Object missing;
 
-    private Boolean ignoreUnmapped;
```

Comment:
https://github.com/elastic/elasticsearch/blob/76fa9023b6378681de34672d2e94227e8b464cfd/core/src/main/java/org/elasticsearch/search/sort/SortParseElement.java#L160

I believe the removal was triggered by this. Probably better discussed in a separate PR?